### PR TITLE
Update binance trade fetcher

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: accounting/cmd/accounting-binance-trade-post-processor/main.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+	sugar.Infow("quotes", "list", quotes)
+
+	retryDelay := c.Duration(retryDelayFlag)
+	attempt := c.Int(attemptFlag)
+	batchSize := 5 // dummy batch size to init fetcher
+
+	binanceFetcher := fetcher.NewFetcher(sugar, binanceClient, retryDelay, attempt, batchSize, binanceStorage, "", marketDataClient)
+	ethTrades, err := binanceStorage.GetNotETHTrades()
```
[the remark this answers is not in the document]
Comment (secmask) [reply accounting/cmd/accounting-binance-trade-post-processor/main.go:120]: accountName seem not use some place.

File: accounting/binance/storage/tradestorage/trade_storage.go [hunk 2]
```diff
@@ -364,3 +365,55 @@ func (bd *BinanceStorage) GetConvertToETHPrice(fromTime, toTime uint64) ([]binan
...
+	}
+	for _, record := range dbResult {
+		arrResult := []binance.TradeHistory{}
+		for _, data := range record.Data {
+			if err := json.Unmarshal(data, &tmp); err != nil {
+				return result, err
+			}
+			arrResult = append(arrResult, tmp)
+		}
+		if _, exist := result[record.Symbol]; exist {
```
Comment (secmask) [inline accounting/binance/storage/tradestorage/trade_storage.go:412]: this if else block can be simple to `result[record.Symbol] = append(result[record.Symbol], arrResult...)`

File: accounting/cmd/accounting-binance-trade-post-processor/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (halink0803) [reply accounting/cmd/accounting-binance-trade-post-processor/main.go]: Yes, accountName is not used in this service. I removed it and init fetcher with account name empty.

File: accounting/cmd/accounting-binance-trade-post-processor/main.go [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+		quotes     = make(map[string]string)
+	)
+	exchangeInfo, err := binanceClient.GetExchangeInfo()
+	if err != nil {
+		return err
+	}
+	tokenPairs = exchangeInfo.Symbols
+	for _, pair := range tokenPairs {
+		if _, exist := quotes[pair.QuoteAsset]; !exist {
+			quotes[pair.QuoteAsset] = pair.QuoteAsset
```
[the remark this answers is not in the document]
Comment (halink0803) [reply accounting/cmd/accounting-binance-trade-post-processor/main.go:110]: I updated it.

**Review by secmask — APPROVED** [review]

**Pull request merged**